compiler/test/stdlib/bytes.test.gr

```diff
@@ -0,0 +1,182 @@
+import Bytes from "bytes";
+import { coerceInt32ToNumber, coerceNumberToInt32 } from "runtime/numbers"
+import { tagSimpleNumber } from "runtime/dataStructures"
```

Comment:
Ideally, we don't import anything from the runtime in tests (unless they're tests for the runtime) since it's not how we want people to use the language. I'll drop some suggestions on how we can replace these.